utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -558,6 +558,11 @@ def create_argument_parser():
            help='A space separated list of targets to cross-compile host '
                 'Swift tools for. Can be used multiple times.')
 
+    option('--cross-compile-deps-path', store_path,
+           help='The path to a directory that contains prebuilt cross-compiled '
+                'library dependencies of the corelibs and other Swift repos, '
+                'such as the libcurl dependency of FoundationNetworking')
```

Comment:
Updated with that last change as described above, squashed all commits so far, and wrote up a commit message with all the changes.

@drexin, would you review?